config/routes.rb

```diff
@@ -17,6 +17,8 @@
   root to: "providers#index"
   get "/organisations", to: redirect("/")
 
+  resources :notifications, path: "/notifications", controller: "notifications" do; end
```

Comment:
`do; end`?
Also looks like you only want `index` and `create`

Reply:
Should be addressed now